OpenRA.Mods.TS/UtilityCommands/ImportGen2MapCommand.cs

```diff
@@ -0,0 +1,443 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2014 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using OpenRA.FileSystem;
+using OpenRA.Mods.Common.FileFormats;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.TS.UtilityCommands
+{
+	class ImportGen2MapCommand : IUtilityCommand
+	{
+		public string Name { get { return "--import-gen2-map"; } }
```

Comment:
I went with the same naming convention at https://github.com/OpenRA/OpenRA/pull/10404 as well then.